README.md

```diff
@@ -6,6 +6,89 @@ connected devices.
 This is not an officially supported Google product; it is currently
 being developed by volunteers on a best-effort basis.
 
+## Contributing
+
+First, read the [contribution guidelines](CONTRIBUTING.md). Ensure you
+understand the code review and community guidelines and have signed a [CLA](https://cla.developers.google.com/). 
```

Comment:
Nit: signed the appropriate CLA 
(vs any CLA)

Reply:
Done!